app/core/utils.coffee

```diff
@@ -24,69 +26,78 @@ countries = [
-  {country: 'sweden', countryCode: 'SE'}
-  {country: 'denmark', countryCode: 'DK'}
-  {country: 'czech-republic', countryCode: 'CZ'}
+  {country: 'belgium', countryCode: 'BE', inEU: true}
+  {country: 'sweden', countryCode: 'SE', inEU: true, ageOfConsent: 13}
+  {country: 'denmark', countryCode: 'DK', inEU: true, ageOfConsent: 13}
+  {country: 'czech-republic', countryCode: 'CZ', inEU: true, ageOfConsent: 13}
   {country: 'hong-kong', countryCode: 'HK'}
-  {country: 'italy', countryCode: 'IT'}
-  {country: 'romania', countryCode: 'RO'}
+  {country: 'italy', countryCode: 'IT', inEU: true}
+  {country: 'romania', countryCode: 'RO', inEU: true}
   {country: 'belarus', countryCode: 'BY'}
   {country: 'norway', countryCode: 'NO'}
   {country: 'philippines', countryCode: 'PH'}
-  {country: 'lithuania', countryCode: 'LT'}
+  {country: 'lithuania', countryCode: 'LT', inEU: true, ageOfConsent: 16}
   {country: 'argentina', countryCode: 'AR'}
   {country: 'malaysia', countryCode: 'MY'}
   {country: 'pakistan', countryCode: 'PK'}
   {country: 'serbia', countryCode: 'RS'}
-  {country: 'greece', countryCode: 'GR'}
-  {country: 'israel', countryCode: 'IL'}
-  {country: 'portugal', countryCode: 'PT'}
-  {country: 'slovakia', countryCode: 'SK'}
-  {country: 'ireland', countryCode: 'IE'}
+  {country: 'greece', countryCode: 'GR', inEU: true, ageOfConsent: 15}
+  {country: 'israel', countryCode: 'IL', inEU: true}
+  {country: 'portugal', countryCode: 'PT', inEU: true}
+  {country: 'slovakia', countryCode: 'SK', inEU: true, ageOfConsent: 16}
+  {country: 'ireland', countryCode: 'IE', inEU: true, ageOfConsent: 13}
   {country: 'switzerland', countryCode: 'CH'}
   {country: 'peru', countryCode: 'PE'}
-  {country: 'bulgaria', countryCode: 'BG'}
+  {country: 'bulgaria', countryCode: 'BG', inEU: true}
   {country: 'venezuela', countryCode: 'VE'}
-  {country: 'austria', countryCode: 'AT'}
-  {country: 'croatia', countryCode: 'HR'}
+  {country: 'austria', countryCode: 'AT', inEU: true, ageOfConsent: 14}
+  {country: 'croatia', countryCode: 'HR', inEU: true}
   {country: 'saudia-arabia', countryCode: 'SA'}
   {country: 'chile', countryCode: 'CL'}
   {country: 'united-arab-emirates', countryCode: 'AE'}
   {country: 'kazakhstan', countryCode: 'KZ'}
-  {country: 'estonia', countryCode: 'EE'}
+  {country: 'estonia', countryCode: 'EE', inEU: true}
   {country: 'iran', countryCode: 'IR'}
   {country: 'egypt', countryCode: 'EG'}
   {country: 'ecuador', countryCode: 'EC'}
-  {country: 'slovenia', countryCode: 'SI'}
+  {country: 'slovenia', countryCode: 'SI', inEU: true}
   {country: 'macedonia', countryCode: 'MK'}
+  {country: 'cyprus', countryCode: 'CY', inEU: true}
+  {country: 'latvia', countryCode: 'LV', inEU: true, ageOfConsent: 13}
+  {country: 'luxembourg', countryCode: 'LU', inEU: true, ageOfConsent: 16}
+  {country: 'malta', countryCode: 'MT', inEU: true}
 ]
 
+inEU = (country) -> !!countries.find((c) => c.country is slugify(country))?.inEU
```

Comment:
Why double negation (!!) is being used for countries.find ?

Reply:
It's a JS trick to convert data to a boolean, in this case an undefined to a boolean value:
https://stackoverflow.com/questions/784929/what-is-the-not-not-operator-in-javascript